doc/whats_new.rst

```diff
@@ -325,6 +325,10 @@ Bug fixes
       (`#7101 <https://github.com/scikit-learn/scikit-learn/pull/7101>`_)
       By `Ibraim Ganiev`_.
 
+    - :func:`silhouette_score` now supports sparse input, and this also fixes
+      examples/text/document_clustering.py. 
+      By `YenChen Lin`_.
```

Comment:
ooops, sorry!

So should I split `silhouette_score` part into enhancement, and example part into bug fix?